.github/workflows/tests.yml

```diff
@@ -33,18 +33,21 @@ jobs:
         conda run -n test make lint
 
   tests:
-    runs-on: ubuntu-latest
     strategy:
         max-parallel: 10
         matrix:
-          python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
+          python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
I think you don't need to chack for python 3.7 and 3.8 anymore, but chack for 3.12 and 3.12 instead.